src/VisualStudio/Core/Def/EditorConfigSettings/Analyzers/View/ColumnDefinitions/AnalyzerCategoryColumnDefinition.cs

```diff
@@ -51,7 +51,7 @@ public AnalyzerCategoryColumnDefinition()
         public override IEntryBucket? CreateBucketForEntry(ITableEntryHandle entry)
         {
             var categoryName = GetCategoryName(entry);
-            return categoryName is not null ? new StringEntryBucket(categoryName) : null;
```

Comment:
It looks like the implementation of the whole class (`AnalyzerCategoryColumnDefinition`) is repeated in the other column definitions. Can we extract a base class?